src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costPerDay = 40;
+  const daysForMinDiscount = 3;
+  const daysForMaxDiscount = 7;
+  const discountPerThreeDays = 20;
```

Comment:
I would rename it
```suggestion
  const minDiscount = 20;
```

Reply:
'Cause you already have var, that discribes "how many days for min/max discount".

Reply:
Agree. You don't want to rename your variables if daysForMinDiscount changes to 4.